torch/functional.py

```diff
@@ -886,22 +887,22 @@ def tensordot(a, b, dims=2, out: Optional[torch.Tensor] = None):  # noqa: F811
 
     When called with a non-negative integer argument :attr:`dims` = :math:`d`, and
     the number of dimensions of :attr:`a` and :attr:`b` is :math:`m` and :math:`n`,
-    respectively, :func:`~torch.tensordot` computes
+    respectively, :func:`torch.linalg.tensordot` computes
 
     .. math::
         r_{i_0,...,i_{m-d}, i_d,...,i_n}
           = \sum_{k_0,...,k_{d-1}} a_{i_0,...,i_{m-d},k_0,...,k_{d-1}} \times b_{k_0,...,k_{d-1}, i_d,...,i_n}.
 
     When called with :attr:`dims` of the list form, the given dimensions will be contracted
     in place of the last :math:`d` of :attr:`a` and the first :math:`d` of :math:`b`. The sizes
-    in these dimensions must match, but :func:`~torch.tensordot` will deal with broadcasted
+    in these dimensions must match, but :func:`torch.linalg.tensordot` will deal with broadcasted
     dimensions.
```

Comment:
Documentation for `tensordot` and implementation should be moved to `linalg/__init__.py`. Leaving here only `tensordot = torch.linalg.tensordot`.